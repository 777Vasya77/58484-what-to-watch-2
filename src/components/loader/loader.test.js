import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '~/components/loader/loader';

describe(`Loader component tests`, () => {
  it(`Component render correctly`, () => {
    const tree = renderer
        .create(
            <Loader />
        ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
