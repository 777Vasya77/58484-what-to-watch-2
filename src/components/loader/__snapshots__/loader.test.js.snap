// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component tests Component render correctly 1`] = `
<div
  style={
    Object {
      "fontSize": "2em",
      "margin": "0 auto",
      "textTransform": "uppercase",
    }
  }
>
  Загрузка...
</div>
`;
